test-case-reporting/index.ts

```diff
@@ -34,6 +34,10 @@ app.get('/test-results/history/:testCaseId', (req, res) => {
   res.send(`Test history for test with name/ID ${testCaseId}`);
 });
 
+app.post('/report', function (req, res) {
+  res.send(`Travis report! The request body is below.\n\n${req.body}`);
```

Comment:
Something like `const report: TravisReport = request.body.report` would set this up for later progress using your type defs

Reply:
Do I rebase this onto the tip of #906 so I have the types?

Reply:
Nevermind, #906 just got merged; I'll pull master then rebase.

Reply:
Done.